comptests/ha/test_ha_shutdown_start_cluster.py

```diff
@@ -127,51 +121,118 @@ def get_ha_node_object(self, master_node_obj) -> LogicalNode:
+        self.deployment_name = resp[1]
+        self.restore_method = common_const.RESTORE_SCALE_REPLICAS
+        self.restore_pod = True
+
+        LOGGER.info("Step 4: Check for publish action event")
+        resp = self.ha_comp_obj.check_string_in_log_file(node_obj, "to component hare",
+                                                         common_const.HA_SHUTDOWN_LOGS[2], lines=4)
+        assert_utils.assert_true(resp[0], "Alert not sent to hare")
+        LOGGER.info("Step 4: Successfully sent action event to hare")
+
+        LOGGER.info("Step 5: Trigger ‘Start Cluster Shutdown’ message to HA.")
+        base_path = os.path.basename(common_const.HA_SHUTDOWN_SIGNAL_PATH)
+        resp = self.ha_comp_obj.shutdown_signal(self.node_master_list[0],
+                                                common_const.HA_SHUTDOWN_SIGNAL_PATH,
+                                                common_const.HA_TMP + '/' + base_path)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 5: Triggered ‘Start Cluster Shutdown’ message to HA. successfully.")
+
+        LOGGER.info("Step 6: Verify fault tolerance log for cluster stop key message.")
+        time.sleep(HA_CFG["common_params"]["20sec_delay"])
+        resp = self.ha_comp_obj.check_string_in_log_file(node_obj, "cluster_stop_key",
+                                                         common_const.HA_SHUTDOWN_LOGS[1])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 6: Verified fault tolerance log for cluster stop key message.")
+
+        LOGGER.info("Step 7: Shutdown entire cortx Cluster")
         resp = self.ha_obj.cortx_stop_cluster(pod_obj=self.master_node_obj)
         assert_utils.assert_true(resp[0], "Error during Stopping cluster")
-        LOGGER.info("Step 1: Shutdown entire cortx Cluster successfully")
+        LOGGER.info("Step 7: Shutdown entire cortx Cluster successfully")
 
-        LOGGER.info("Step 2:Verify all HA logs for SIGTERM alert message")
+        LOGGER.info("Step 8:Verify all HA logs for SIGTERM alert message")
         for log in common_const.HA_SHUTDOWN_LOGS:
-            resp = HAK8SCompLib.check_string_in_log_file(node_obj, "SIGTERM", log, lines=18)
+            resp = self.ha_comp_obj.check_string_in_log_file(node_obj, "SIGTERM", log, lines=18)
             assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 2:Verified all HA logs for SIGTERM alert message %s",
+        LOGGER.info("Step 8:Verified all HA logs for SIGTERM alert message %s",
                     common_const.HA_SHUTDOWN_LOGS)
-        self.start_cluster = True
 
-        LOGGER.info("Completed: Stopped Cluster - Shutdown cluster, verified HA alerts log SIGTERM")
+        LOGGER.info("Step 9: Start the cluster")
+        resp = self.ha_obj.restart_cluster(self.node_master_list[0])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 9: Successfully started the cluster")
```

Comment:
Can you add self.restore_pod = False in this TC, as I see self.restore_pod = True and in teardown after the start it will try to scale up again. Also check this in all TCs.